### PR TITLE
feat(ado-auth-part-2): Add ADO authentication inputs

### PR DESCRIPTION
**Pull request by ThanyaLeif**
#### Details

This PR adds the ADO authentication inputs `serviceAccountName`, `serviceAccountPassword`, and `authType` added in the latest released version of `accessibility-scan-package` for the new authentication flow.

##### Motivation

Feature work 

##### Context

This new authentication flow will exclusively affect the ADO Extension and currently only supports AAD type authentication; we plan to include more types of authentication in a future feature.

#### Pull request checklist

- [x] Addresses an existing issue: Fixes [1970882](https://dev.azure.com/mseng/1ES/_workitems/edit/1970882/)
- [x] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`yarn precheckin`)
- [ ] Described how this PR impacts both the ADO extension and the GitHub action

### CODE REVIEW

**Review by brocktaylor7 — APPROVED** [review]
LGTM with one small suggestion :)

File: packages/ado-extension/task.json [hunk 1]
```diff
@@ -70,6 +70,30 @@
             "helpMarkDown": "The hosted URL to scan/crawl for accessibility issues.",
             "visibleRule": "hostingMode = dynamicSite"
         },
+        {
+            "name": "serviceAccountName",
+            "type": "string",
+            "label": "User name",
```
Comment (katydecorah) [inline packages/ado-extension/task.json:76]: Do you think we should update this label value to be "Service account name?" I noticed that we swapped username out everywhere else.

File: packages/ado-extension/task.json [hunk 2]
```diff
@@ -70,6 +70,30 @@
...
+            "required": false,
+            "helpMarkDown": "User credentials"
```
[the remark this answers is not in the document]
Comment (katydecorah) [reply packages/ado-extension/task.json:78]: The CLI has some descriptions that we could considering reusing here: https://github.com/microsoft/accessibility-insights-service/blob/edf2e3d4bc37b5810dbb165650ce70002cc46a05/packages/cli/src/cli.ts#L132-L142

**Review by katydecorah — APPROVED** [review]
Looks good to me! I left one small suggestion for a label value.

**Pull request merged**